src/routes/rasa.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { Switch, Route, useRouteMatch } from 'react-router-dom'
+import queryString from 'query-string'
+
+import RasaPage from '../pages/rasa'
+
+function parseQueryString() {
+  const hash = window.location.hash
+  return queryString.parse(hash.substr(hash.indexOf('?')))
+}
+
+function getStringParameter(parameter) {
+  const uid = parseQueryString()[parameter]
+  if (typeof uid === 'string') return uid
+  return undefined
+}
+
+function getTimezone() {
+  const defaultDateTimeFormat = new Intl.DateTimeFormat()
+  return defaultDateTimeFormat.resolvedOptions().timeZone
+}
+
+function RouteRasa() {
+  let { path } = useRouteMatch()
+
+  const renderDefault = props => {
+    const timezone = getTimezone()
+    return <RasaPage {...props} timezone={timezone} />
```

Comment:
Like said above we should pass the `initPayload` here. We could also pass those props (`timezone` and `userId`) if need be, but separately.